src/components/datagrid/column_selector.tsx

```diff
@@ -118,15 +142,18 @@ export const useColumnSelector = (
               size="xs"
               flush="left"
               onClick={() => setVisibleColumns(sortedColumns)}>
-              Show all
+              <EuiI18n
+                token="euiColumnSelector.selectAll"
+                default="Select all"
```

Comment:
This should be the opposite of "hide"

```suggestion
                default="Show all"
```